marge/job.py

```diff
@@ -44,8 +44,8 @@ def ensure_mergeable_mr(self, merge_request):
         log.info('Ensuring MR !%s is mergeable', merge_request.iid)
         log.debug('Ensuring MR %r is mergeable', merge_request)
 
-        if merge_request.work_in_progress:
-            raise CannotMerge("Sorry, I can't merge requests marked as Work-In-Progress!")
+        if merge_request.work_in_progress or merge_request.draft:
+            raise CannotMerge("Sorry, I can't merge requests marked as Draft!")
```

Comment:
Hmm... do both of these concepts exist in GitLab? :thinking_face:

Reply:
Hmm.. I thought `work_in_progress` didn't exist any more, because I had a feeling this doesn't work as it should any more :thinking: But it looks like both fields exist. But maybe `work_in_progress` will eventually be removed :shrug: 

https://docs.gitlab.com/ee/api/merge_requests.html#list-project-merge-requests 

![image](https://user-images.githubusercontent.com/1417619/209080667-c402b200-0b69-48ee-90a3-51bc010a7bba.png)

Reply:
The tests are failing though :grin: Probably need to check that the field exists too.

Reply:
Fixing like this instead:

```diff
diff --git marge/job.py marge/job.py
index 9c3d365..ba3841a 100644
--- marge/job.py
+++ marge/job.py
@@ -44,7 +44,7 @@ class MergeJob:
         log.info('Ensuring MR !%s is mergeable', merge_request.iid)
         log.debug('Ensuring MR %r is mergeable', merge_request)
 
-        if merge_request.work_in_progress or merge_request.draft:
+        if merge_request.work_in_progress:
             raise CannotMerge("Sorry, I can't merge requests marked as Draft!")
 
         if merge_request.squash and self._options.requests_commit_tagging:
diff --git marge/merge_request.py marge/merge_request.py
index 0dee208..5ce89dc 100644
--- marge/merge_request.py
+++ marge/merge_request.py
@@ -141,7 +141,7 @@ class MergeRequest(gitlab.Resource):
 
     @property
     def work_in_progress(self):
-        return self.info['work_in_progress']
+        return self.info['work_in_progress'] or ('draft' in self.info and self.info['draft'])
 
     @property
     def approved_by(self):
```